config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -25,7 +25,6 @@ presubmits:
         - --build=quick
         - --cluster=
         - --down=false
-        - --extract=local
```

Comment:
this job doesn't work, but it's manual, non-blocking, and we don't want any remaining old references to get copy-pasted.